aspnetcore/security/authentication/google-logins.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Google external login setup | Microsoft Docs
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 11/2/2016
+ms.topic: article
+ms.assetid: 8BA389D6-0911-4415-A818-C7B3F5B5CC8D
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: security/authentication/google-logins
+---
+# Configuring Google authentication
+
+<a name=security-authentication-google-logins></a>
+
+This tutorial shows you how to enable your users to login with their Google+ account using a sample ASP.NET Core project created in the [previous section](sociallogins.md). We start by following the [official steps](https://developers.google.com/identity/sign-in/web/devconsole-project) to create a new app in Google API Console.
+
+## Creating the app in Google API Console
+
+* Navigate to [https://console.developers.google.com/projectselector/apis/library](https://console.developers.google.com/projectselector/apis/library) and sign in. If you don't already have a Google account, use the *[Create account](https://accounts.google.com/SignUpWithoutGmail?service=cloudconsole&continue=https%3A%2F%2Fconsole.developers.google.com%2Fprojectselector%2Fapis%2Flibrary&ltmpl=api)* link to create one:
+
+![image](sociallogins/_static/GoogleConsoleLogin.png)
+
+* You will be redirected to API Manager Library page:
+
+![image](sociallogins/_static/GoogleConsoleSwitchboard.png)
+
+* Tap **Create a project** and enter your application name:
+
+![image](sociallogins/_static/GoogleConsoleNewProj.png)
+
+* You will be redirected back to the Library page allowing you to choose features your new app will support. <a name="enable-googleplus">Find *Google+ API* in the list</a> and click on its link to add the API feature:
+
+![image](sociallogins/_static/GoogleConsoleChooseApi.png)
+
+* The page for the newly added API will be displayed. Tap **Enable** to add Google+ login feature to your app:
+
+![image](sociallogins/_static/GoogleConsoleEnableApi.png)
+
+* After enabling the API, tap **Go to Credentials** to configure the secrets:
+
+![image](sociallogins/_static/GoogleConsoleGoCredentials.png)
+
+* Choose:
+   * **Google+ API**
+   * **Web server (e.g. node.js, Tomcat)**, and
+   * **User data**:
+
+![image](sociallogins/_static/GoogleConsoleChooseCred.png)
+
+* Tap **What credentials do I need?** which will take you to the second step of app configuration:
+
+![image](sociallogins/_static/GoogleConsoleCreateClient.png)
+
+* Because we are creating a Google+ project with just one feature (logins), we can enter the same **Name** for the OAuth 2.0 client ID as the one we used for the project.
+
+* Enter your current site URL with *signin-google* appended into the **Authorized redirect URIs** field. For example, `https://localhost:44320/signin-google`.
+
+* When deploying the site you'll need to register a new public url.
```

Comment:
comment to previous step rather than separate step

Also indent the Note two spaces, as it pertains to this step.

* step text

  > [!NOTE]
  > Note that pertains to the step